### PR TITLE
Add metrics for sealingstatus

### PR DESCRIPTION
**Pull request by cviecco**

### CODE REVIEW

File: cmd/cloud-gate/main.go [hunk 1]
```diff
@@ -84,6 +86,21 @@ func main() {
 	}
 	webServer.AddHtmlWriter(logger)
 
+	//
```
Comment (rgooch) [inline cmd/cloud-gate/main.go]: Please remove empty comment line (and blank lines)

File: cmd/cloud-gate/main.go [hunk 2]
```diff
@@ -84,6 +86,21 @@ func main() {
...
+	isReadyMetric := prometheus.NewGaugeFunc(
+		prometheus.GaugeOpts{
+			Name: "cloudgate_isReady",
+			Help: "coudgate is unsealed and ready to accept connections",
```
Comment (rgooch) [inline cmd/cloud-gate/main.go]: s/coudgate/cloudgate/

File: lib/userinfo/ldap/impl.go [hunk 3]
```diff
@@ -2,14 +2,45 @@ package ldap
...
+			Name:       "cloudgate_ldap_userinfo_check_duration_seconds",
+			Help:       "LDAP Dependency latency",
+			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001},
+		},
+		[]string{"target"},
+	)
+	userinfoLDAPAttempt = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "cloudgate_ldap_userinfo_attempt_counter",
+			Help: "test help",
```
Comment (rgooch) [inline lib/userinfo/ldap/impl.go]: Please add a real comment.

File: lib/userinfo/ldap/impl.go [hunk 4]
```diff
@@ -2,14 +2,45 @@ package ldap
...
+		},
+	)
+	userinfoLDAPSuccess = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "cloudgate_ldap_userinfo_success_counter",
+			Help: "test help",
```
Comment (rgooch) [inline lib/userinfo/ldap/impl.go]: Ditto.

File: cmd/cloud-gate/main.go (hunk 1 above)
Comment (cviecco) [reply cmd/cloud-gate/main.go]: ack, done

**Pull request merged**